### DISCUSSION TITLE
React refresh caused CSP block 'unsafe-eval'

### DISCUSSION THREAD

**Discussion by seta-hainguyen** [vercel/next.js#28019] [Help]
### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

14.17.3

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

Express custom server

### Describe the Bug

When CSP for `script-src` is enabled, it throws the errors when running the app
![image](https://user-images.githubusercontent.com/64835563/129187683-c3cb52ff-4442-4a6c-bbf4-4cf3602d01e0.png)

### Expected Behavior

Should pass this CSP's check

### To Reproduce

I think you can reproduce that by running normal command like next start, etc

**Comment by jscheinhorn** [comment]
I saw the same thing in stage env and it was due to istanbul plugin
